src/Tester/FSharpGrainTests.cs

```diff
@@ -14,20 +12,9 @@ namespace UnitTests.General
     /// Unit tests for grains implementing generic interfaces
     /// </summary>
     [TestClass]
-    public class FSharpGrainTests : UnitTestSiloHost
+    public class FSharpGrainTests : UnitTestSiloHostEnsureDefaultStartedSecondaryOff
```

Comment:
But the original version of this test particular was only using a single silo (Primary, but no Secondary)

``` c#
public FSharpGrainTests()
    : base(new TestingSiloOptions { StartPrimary = true, StartSecondary = false })
{
}
 ```

Isn't that why you used the `SecondaryOff` part of the name `UnitTestSiloHostEnsureDefaultStartedSecondaryOff` ?

I think the `UnitTestSiloHostEnsureDefaultStarted` name for the "normal" (2-silo) case is ok, 
but `UnitTestSiloHostEnsureDefaultStartedSecondaryOff` seems to bury the headline that this test case is only using single-silo.
But i do take your point that the other part of the name is the "Defaults" part to signify using the baseline "default" config.

How about something link this:
`UnitTestSiloHostEnsureDefaultSingleSiloStarted`

BTW, i don't think we should get too hung up on finding the perfect name here, so this is P2 issues in my book - way more important to get this change set merged than to rat hole on names.